src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+html {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 400;
+}
+
+.search {
+  background: rgb(255, 255, 255);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.containerBig {
+  margin-top: 20px;
+}
+
+.containerSmall {
+  margin-top: 20px;
+}
```

Comment:
Don't duplicate styles. Check 2nd point from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)